lib/src/resources/movies_repository.dart

```diff
@@ -5,5 +5,10 @@ import '../models/movie_model.dart';
 class MoviesRepository {
```

Comment:
This repository should also be implementing an interface

Reply:
fixed